script.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-use-before-define */
+const saveBtn = document.querySelector('#addBtn');// eslint-disable-line no-unused-vars
+const bookListDiv = document.querySelector('#bookList');
+dataStorage(JSON.parse(localStorage.getItem('books'))); // eslint-disable-line no-undef
+function addBook() {
+  const bookList = JSON.parse(localStorage.getItem('books')) === null ? [] : JSON.parse(localStorage.getItem('books'));
+  const book = {
+    title: document.querySelector('#inputTitle').value,
+    author: document.querySelector('#inputAuthor').value,
+  };
+
+  bookList.push(book);
+  localStorage.setItem('books', JSON.stringify(bookList));
+  dataStorage(bookList); // eslint-disable-line no-undef
+}
+
+addBtn.addEventListener('click', addBook); // eslint-disable-line no-undef
```

Comment:
- [ ] I would not recommend disabling linters, instead fix all the linters by improving the code.